### ISSUE TITLE
Not showing correct plan details on header

### ISSUE THREAD

**Issue by rr-krupesh-savaliya** [ahmadnassri/action-terraform-report#44]
**Issue:** Show the wrong plan in header details.

<img width="292" alt="Screenshot 2022-09-15 at 12 09 44 AM" src="https://user-images.githubusercontent.com/105941782/190236093-61de2ca7-045b-41e4-8b1e-18b2a94fd77c.png">

As you can see above, add value is **3**, and destroy value is **4**. but the actual plan details are as per below.

<img width="359" alt="Screenshot 2022-09-15 at 12 11 30 AM" src="https://user-images.githubusercontent.com/105941782/190236837-71f97277-072a-4598-add6-abf0a33c8348.png">

Thanks

**Comment by ahmadnassri** [comment]
interesting... can you grab the json object and see what it says? 

this value comes straight from the json data generated by the plan

**Comment by rr-krupesh-savaliya** [comment]
I can't show all terraform plan. but In the plan, these 3 actions are included.

<img width="555" alt="Screenshot 2022-09-16 at 12 25 07 AM" src="https://user-images.githubusercontent.com/105941782/190486730-207e5704-d68f-4052-bfbb-7a8668ba8f4f.png">

Action Counts:

create(+) = 3
destroy and create(-/+) = 4
create and destroy(+/-) = 1

So, total `7 to add and 5 to destroy.`

This will help you to reproduce the issue.

Thanks

**Comment by ahmadnassri** [comment]
you don't need to share the whole plan (I wouldn't expect that) 

the code reads the count values from the JSON data itself (see here: https://github.com/ahmadnassri/action-terraform-report/blob/96c46fe871e2f7ef1142307c924fd076b69de12b/lib/report.js#L14)

so I was curious if the json data is in fact showing the same count, and thus, something is wrong the the JSON generation of the plan (and we can report a bug to hashicorp) ... or something else happened between the plan json generation and the action being executed (which we can chase further)

**Comment by ahmadnassri** [comment]
I think I found the issue, it's here: https://github.com/ahmadnassri/node-terraform-unidiff/blob/a54705ee0aa6c2751cde40c0b23f08678c082f85/index.js#L38-L43

and according to the terraform docs: https://www.terraform.io/internals/json-format#change-representation

```
Valid actions values are:
  //    ["no-op"]
  //    ["create"]
  //    ["read"]
  //    ["update"]
  //    ["delete", "create"]
  //    ["create", "delete"]
  //    ["delete"]
```

I clearly wasn't taking into account the two scenarios of `["delete", "create"]` and `["create", "delete"]` ...

fix will be up shortly, thanks for your help!

**Comment by ahmadnassri** [comment]
fix should be released in [v3.1.5](https://github.com/ahmadnassri/action-terraform-report/releases/tag/v3.1.5)

please let me know if it's working for you.
